src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: $card-width ;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $white-color;
+  padding: 32px 16px 16px;
+  transition-property: transform;
+  transition-duration: $transition-duration;
+  margin: 0;
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+  font-family: $main-font;
+
+  &__title {
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    transition-duration: $transition-duration;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: $card-hover-title-color;
+    }
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-reviews {
+      color: $main-color;
+      line-height: 14px;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+    }
+  }
+
+  &__buy {
+    display: inline;
+    // display: block;
```

Comment:
It's better to do this button an `a`(link) element. And you don't need to change the display property to inline. Also, if it's a link you can open in a new tab the product that you want to check